tests/Backend/CommonPart1/TableWithConfigurationLoadFromWorkspaceTest.php

```diff
@@ -98,7 +98,7 @@ public function testLoadFromWorkspaceToTable(): void
         "description": ""
       },
       {
-        "sql": "CREATE TABLE {{ id(schemaName) }}.{{ id(tableName ~ rand ~ '_tmp') }} WITH (DISTRIBUTION=ROUND_ROBIN,CLUSTERED COLUMNSTORE INDEX) AS SELECT a.[id],a.[NAME] FROM (SELECT COALESCE([id], '') AS [id],COALESCE([NAME], '') AS [NAME], ROW_NUMBER() OVER (PARTITION BY [id] ORDER BY [id]) AS \"_row_number_\" FROM {{ id(stageSchemaName) }}.{{ id(stageTableName) }}) AS a WHERE a.\"_row_number_\" = 1",
+        "sql": "CREATE TABLE {{ id(schemaName) }}.{{ id(tableName ~ rand ~ '_tmp') }} WITH (DISTRIBUTION=ROUND_ROBIN,CLUSTERED COLUMNSTORE INDEX) AS SELECT a.[id],a.[NAME],a.[_timestamp] FROM (SELECT COALESCE([id], '') AS [id],COALESCE([NAME], '') AS [NAME],CAST({{ timestamp }} as DATETIME2) AS [_timestamp], ROW_NUMBER() OVER (PARTITION BY [id] ORDER BY [id]) AS \\"_row_number_\\" FROM {{ id(stageSchemaName) }}.{{ id(stageTableName) }}) AS a WHERE a.\\"_row_number_\\" = 1",
```

Comment:
\"_row_number_\" vs \\"_row_number_\\"?

Reply:
Jo jo, ale nerozumím, jak to mohlo fungovat. Protože... https://3v4l.org/okOSC6

Reply:
🤷 ...magic 🧙?